index.html

```diff
@@ -58,9 +58,12 @@
       <p>footerFooterFooter</p>
     </footer>
 
-    <script src='data/blogData.js'></script>
+    <script src='html5sql.js'></script>
+    <script src='webDB.js'></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
     <script src='article.js'></script>
     <script src='blog.js'></script>
     <script src='index.js'></script>
+    <script src='edit_articles.js'</script>
```

Comment:
Does this really need to be in your index.html since it seems to only be used in edit_articles.html? I think this is one reason your page is loading slow. I'm seeing errors in your console related to this.